src/msbuild/DNNE.targets

```diff
@@ -47,21 +47,34 @@ DNNE.targets
     <DnneNativeBinaryExt Condition="$([MSBuild]::IsOsPlatform('OSX'))">.dylib</DnneNativeBinaryExt>
     <DnneNativeBinaryExt Condition="'$(DnneNativeBinaryExt)' == '' OR $([MSBuild]::IsOsPlatform('Linux'))">.so</DnneNativeBinaryExt>
 
-    <!-- Compute details about TFM -->
-    <DnneIsNetFramework>false</DnneIsNetFramework>
-    <DnneIsNetFramework Condition="$(TargetFramework.StartsWith('net4'))">true</DnneIsNetFramework>
-    <DnneSupportedTFM>true</DnneSupportedTFM>
-    <DnneSupportedTFM Condition="!$([MSBuild]::IsOsPlatform('Windows')) AND '$(DnneIsNetFramework)' == 'true'">false</DnneSupportedTFM>
```

Comment:
The `DnneSupportedTFM` is used in the `DnneBuildNativeExports` target's condition clause. This should be kept.